components/snap_recovery/src/init_cluster.rs

```diff
@@ -85,6 +85,14 @@ pub fn enter_snap_recovery_mode(config: &mut TikvConfig) {
     config.raft_store.max_snapshot_file_raw_size = ReadableSize::gb(1);
     config.raft_store.hibernate_regions = false;
 
+    // Disable prevote so it is possible to regenerate leaders.
+    config.raft_store.prevote = false;
+    // Due to we have increased the election tick to inf, once there is a leader,
+    // the follower will believe it holds an eternal lease. So, once the leader
+    // reboots, the followers will reject to vote for it again.
```

Comment:
where the code that followers reject to vote

Reply:
@Connor1996 Because election tick is inf so leader lease never expires.